src/js/components/dashboard/widgets/baseWidget.js

```diff
@@ -4,9 +4,11 @@ export const BaseWidget = ({ className = '', header, innerRef, main, onClick })
   <div className={`widget ${className}`} onClick={onClick} ref={ref => (innerRef ? (innerRef.current = ref) : null)}>
     {!!header && <div className="flexbox widgetHeader">{header}</div>}
     {React.isValidElement(main) ? (
-      <div className="widgetMainContent">{main}</div>
+      <div style={{ justifyContent: 'space-evenly' }} className="widgetMainContent">
```

Comment:
https://github.com/mendersoftware/gui/blob/master/src/js/components/dashboard/widgets/actionabledevices.js#L85 <- might be the proper place to set this

Reply:
Ohh, I see it now, thx :)